skyportal/handlers/api/candidate.py

```diff
@@ -1,4 +1,6 @@
 import arrow
+from copy import copy
+import datetime
```

Comment:
As per PEP8, import of python builtins should come before 3rd party imports (arrow)

Reply:
Done